test/test_bmark.jl

```diff
@@ -13,8 +13,8 @@ function test_bmark()
     callable = CallableSolver()
     stats = solve_problems(dummy_solver, problems)
     @test stats isa DataFrame
-    stats = solve_problems(dummy_solver, problems, reset_solver=false)
-    stats = solve_problems(dummy_solver, problems, reset_solver=true)
+    stats = solve_problems(dummy_solver, problems, reset_problem=false)
+    stats = solve_problems(dummy_solver, problems, reset_problem=true)
```

Comment:
How was this passing before?

Reply:
`solve_problems` accept `kwargs...` to pass to the solver inside a `try` block, so it passed the tests even though it failed.